dw-select-base-dialog.js

```diff
@@ -546,7 +546,7 @@ export class DwSelectBaseDialog extends DwCompositeDialog {
       <lit-virtualizer
         .items=${this._items}
         .renderItem=${(item, index) =>
-          this.renderItem ? this.renderItem(item) : this._defaultTemplate(item, index)}
+          this.renderItem ? this.renderItem(index, item) : this._defaultTemplate(item, index)}
```

Comment:
this seems a backward incompatible change, If a new parameter `index` is required, it must be 2nd parameter.